scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/core/service/impl/FileSystemService.java

```diff
@@ -32,10 +31,10 @@ public FileSystemService() {
      *
      * @param id       The id of the dataset.
      * @param username The username of the dataset.
-     * @throws DatasetStorageException The exception thrown if the directory could not be created.
+     * @throws FileSystemStorageException The exception thrown if the directory could not be created.
      */
     @Override
-    public void addDatasetDirectories(String id, String username) throws DatasetStorageException {
+    public void addDatasetDirectories(String id, String username) throws FileSystemStorageException {
```

Comment:
this doesn't necessarily have to be a dataset directory, right? This method would create a new directory in a user's folder no matter what it is used for. I would rename the method.